docs/pages/deploy-a-cluster/deployments/aws-active-passive-deployment.mdx

```diff
@@ -0,0 +1,112 @@
+- High-availability Auto Scaling group of Proxy Service pods deployed across multiple regions
+
+### Disadvantages of this deployment architecture
+
+- Long-term cost may be a prohibitive factor for some organizations and can increase total cost of ownership
+  (TCO) throughout the system's lifetime.
+- Deploying and maintaining the added layer of regional components takes more engineering effort.
+- This architecture is more technically complex to deploy than a single region Teleport cluster or simple disaster recovery.
+
+### Configuration Example in Kubernetes with the `teleport-cluster` Helm Chart:
+
+## Prerequisites
+
+(!docs/pages/kubernetes-access/helm/includes/teleport-cluster-prereqs.mdx!)
+
+## Step 1/6. Install Helm
+
+(!docs/pages/kubernetes-access/helm/includes/teleport-cluster-install.mdx!)
+
+## Step 2/6. Add the Teleport Helm chart repository
+
+(!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
+
+## Step 3/6. Set up Teleport Cluster in Primary Region (In this example us-west-1 is the Primary Region):
+- Running an HA Teleport cluster using [AWS, EKS, Helm](https://goteleport.com/docs/deploy-a-cluster/helm-deployments/aws/)
+- Configure [Single Sign-on](https://goteleport.com/docs/access-controls/sso/) and test your access to the Teleport cluster.
+
+## Step 4/6. Setup DynamoDB & S3 Global replication to Secondary Region (In this example us-east-1 is the Secondary Region):
+- Check the Key Components section of this document for DynamoDB Global replication and S3 Replication.
+- Post successful completion of Replication DynamoDB and S3 proceed to Step 5/6.
+
+## Step 5/6. Set up Teleport cluster in Secondary Region same as Step 3/6:
+- We will use the same `aws-values.yaml` & `aws-issuer.yaml` config files from the Primary region cluster to install the `teleport-cluster` Helm chart in the Secondary Region (`us-east-1`), by changing the `clusterName`.
+  So that it will use same backend as defined in Primary Region (us-west-1) for DynamoDB and S3 buckets. 
+  Once the Teleport Cluster in Secondary Region (us-east-1) is ready, you can check the SSO access by logging in via Secondary region (us-east-1) UI/tsh
+  Configuration. SSO login will work fine from both the Primary (us-west-1) and Secondary (us-east-1) region Teleport clusters.
+
+## Step 6/6. Configure Proxy Peering:
+In this deployment architecture, [Proxy Peering](../../architecture/proxy-peering.mdx) is used to restrict the number of connections made from 
+resources to proxies in the Teleport Cluster.
+ 
+### Auth Service Proxy Peering configuration 
+
+The Teleport Auth Service must be configured to use the `proxy_peering` tunnel strategy as shown in the example below:
+
+```
+auth_service:
+ ...
+ tunnel_strategy:
+  type: proxy_peering
+  agent_connection_count: 2
+```
+Reference the [Auth Server configuration](../../reference/config.mdx#auth-service) reference page 
+for additional settings.
```

Comment:
Add more detail why the value 2 was selected, indicating that this represents the total number of auth pods in the active region and that no more than 2 are supported by DynamoDB